config-editor/config-editor-ui/src/app/services/editor.service.ts

```diff
@@ -21,9 +21,9 @@ export class ServiceContext {
   adminSchema?: AdminSchemaService;
   configStore: ConfigStoreService;
   serviceName: string;
-  testSpecificationSchema?: JSONSchema7;
   adminMode: boolean;
   searchHistoryService?: SearchHistoryService;
+  testConfigSpec?: TestConfigSpec[];
```

Comment:
i think this should be redesigned and store test list of specification per use case eg test, test case, release